gadget-container/gadgets/seccomp/bpf/seccomp.c

```diff
@@ -0,0 +1,94 @@
+// SPDX-License-Identifier: GPL-2.0
+/* Copyright (c) 2021 The Inspektor Gadget authors */
+
+/* This BPF program uses the GPL-restricted function bpf_probe_read*().
+ */
+
+#include "vmlinux.h"
+
+#include <bpf/bpf_helpers.h>
+#include <bpf/bpf_core_read.h>
+#include <bpf/bpf_tracing.h>
+
+#include "common.h"
+
+#define TASK_COMM_LEN 16
+#define TS_COMPAT 0x0002
+
+#define nr_seccomp 317
```

Comment:
At the moment, we only support x86_64, so I think it is ok.

I think a global eBPF variable is not worth it because the eBPF program would need to be recompiled anyway for other architectures (e.g. vmlinux.h is from x86_64).

I'll add a comment.